code/__defines/unit_testing.dm

```diff
@@ -7,4 +7,4 @@
  *   Should you wish to edit set UNIT_TEST to 1 like so:
  *   #define UNIT_TEST 1
  */
-#define UNIT_TEST 0
+#define UNIT_TEST 1
```

Comment:
This should never been commited

Reply:
Oh right, forgot to take it out. Thank you